lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -15407,3 +15407,21 @@ msgstr "Le port ne peut être compris qu'entre 0 et 65535 inclus"
 
 msgid "No results found"
 msgstr "Aucun résultat trouvé"
+
+msgid "Save filter"
+msgstr "Sauvegarder le filtre"
+
+msgid "Save as new"
+msgstr "Sauvegarder comme nouveau"
```

Comment:
seems to be a bit weird to display this.
Could we use `Enregistrer sous` like it is used in other softwares ?
```suggestion
msgstr "Sauvegarder comme nouveau"
```